site/content/overview/custom-policies.md

```diff
@@ -0,0 +1,159 @@
+
+Settings defined in a policy attached to an object in this hierarchy may be inherited by the resources below it. For example, the settings defined in a policy attached to a Gateway may be inherited by all the HTTPRoutes attached to that Gateway.
+
+Settings in an Inherited Policy can be Defaults or Overrides. Defaults set the default value for something and can be overridden by policies on a lower object. Overrides cannot be overridden by lower objects.
+All settings in NGINX Gateway Fabric Inherited Policies are Defaults.
+
+Default values are given precedence from the bottom up. Therefore, a policy setting attached to a Backend will have the highest precedence over the one attached to higher objects.
+
+The following diagram shows how Inherited Policies work in NGINX Gateway Fabric using a fictional retry policy:
+
+{{<img src="img/inherited-policy-attachment.png" alt="Inherited Policy Attachment">}}
+
+There are three policies defined:
+
+- `dev-policy` that targets the `dev` Gateway
+- `baz-policy` that targets the `baz` HTTPRoute
+- `foo-policy` that targets the `foo` HTTPRoute
+
+The settings in `dev-policy` affect the `dev` Gateway and are inherited by all the HTTPRoutes attached to `dev`.
+The `baz-policy` and `foo-policy` are attached to the `baz` and `foo` HTTPRoutes. Since HTTPRoutes are lower than Gateways in the hierarchy, the settings defined in them override those in the `dev` policy.
+Since the `foo-policy` only defines the `retries` setting, it still inherits the `timeout` setting from `dev-policy`.
+The `bar` HTTPRoute has no policy attached to it and inherits all the settings from `dev-policy`.
+
+## Merging Policies
+
+With some NGINX Gateway Fabric Policies, it is possible to create multiple policies that target the same resource as long as the fields in those policies do not conflict.
+
+For example, consider the following fictional policies:
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha1
+kind: ExamplePolicy
+metadata:
+  name: retries
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: foo
+  retries: 10
+```
+
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha1
+kind: ExamplePolicy
+metadata:
+  name: timeout
+spec:
+  targetRef:
+    kind: HTTPRoute
+    name: foo
+  timeout: 60s
+```
+
+The `retries` ExamplePolicy defines the number of retries for the `foo` HTTPRoute, and the `timeout` ExamplePolicy defines the timeout for the `foo` HTTPRoute.
+NGINX Gateway Fabric will merge the fields defined in the policies and apply the following settings to the `foo` HTTPRoute:
+
+```yaml
+retries: 10
+timeout: 60s
+```
+
+However, if both policies had the `retries` field set, then the policies cannot be merged. In this case, NGINX Gateway Fabric will choose which policy to configure based on the following criteria (continuing on ties):
+
+1. The oldest policy by creation timestamp
+2. The policy appearing first in alphabetical order by "{namespace}/{name}"
```

Comment:
```suggestion
1. The policy appearing first in alphabetical order by "{namespace}/{name}"
```

Hugo will automatically enumerate list items starting with 1. This feels unnecessary for 2 items, but means ordered lists of arbitrary size can be changed without requiring each item to be manually numbered.